frontend/lib/graphql/configured_graphql_provider.dart

```diff
@@ -1,14 +1,57 @@
+import 'dart:io';
+
 import 'package:ehrenamtskarte/configuration/configuration.dart';
+import 'package:fk_user_agent/fk_user_agent.dart';
 import 'package:ehrenamtskarte/sentry.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:graphql_flutter/graphql_flutter.dart';
 
-class ConfiguredGraphQlProvider extends StatelessWidget {
+class ConfiguredGraphQlProvider extends StatefulWidget {
   final Widget child;
 
   const ConfiguredGraphQlProvider({super.key, required this.child});
 
+  @override
+  State<StatefulWidget> createState() => ConfiguredGraphQlProviderState();
+}
+
+class ConfiguredGraphQlProviderState extends State<ConfiguredGraphQlProvider> {
```

Comment:
What are we using this whole thing for? What events/things are we tracking from the frontend? How is the tracking with matomo connected to graphql?